packages/agent/src/app.test.ts

```diff
@@ -155,60 +161,6 @@ describe('App', () => {
     mockServer2.stop();
   });
 
-  test('Empty endpoint URL', async () => {
```

Comment:
Removed this test b/c the resource validator prevents Endpoints w/ empty addresses from being created